assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.js

```diff
@@ -63,18 +67,25 @@ function ConversionReportingNotificationCTAWidget( { Widget, WidgetNull } ) {
 	const shouldShowInitialCalloutForTailoredMetrics =
 		! hasUserPickedMetrics?.length &&
 		isUserInputCompleted &&
-		haveConversionReportingEventsForTailoredMetrics &&
-		! keyMetricSettings?.includeConversionTailoredMetrics;
+		haveConversionReportingEventsForTailoredMetrics;
```

Comment:
Since logic around lost events and persisting ACR metrics has been changed, only this check is needed now